README.md

```diff
@@ -1,3 +1,19 @@
+---
+### Repository sponsored by [Anvil](www.useanvil.com/developers) 
```

Comment:
I think we should remove the 'sponsored by' text. It could just be 'What is Anvil?" or something like that

Reply:
I'd also put this section below the first section of the readme, above the build section

Reply:
I would like to keep the "sponsored by" section on all of our public repos that are not an Anvil client and not Anvil demo code. So SpectaQL, Microfiber etc. 

If we are going to market Anvil, we need to *market* Anvil.

Reply:
Can move it down below the first section, do all of our repos follow this pattern in the readme?